otx/algorithms/detection/tasks/__init__.py

```diff
@@ -27,4 +29,5 @@
     "DetectionNNCFTask",
     "OpenVINODetectionTask",
     "MPADetection",
+    "OTXModels",
```

Comment:
is it used in other code by `import *`?

Reply:
This is as described in the comments above. import to register the model and backbone to the registry.